server/routerlicious/kubernetes/routerlicious/values.yaml

```diff
@@ -171,3 +171,6 @@ error:
 lumberjack:
   options:
     enableGlobalTelemetryContext: true
+
+config:
```

Comment:
We might also need to make similar changes to the Historian and GitRest configmaps:

https://github.com/microsoft/FluidFramework/blob/main/server/charts/historian/templates/gitrest-configmap.yaml

https://github.com/microsoft/FluidFramework/blob/main/server/charts/historian/templates/historian-configmap.yaml

And their values.yaml:
https://github.com/microsoft/FluidFramework/blob/main/server/charts/historian/values.yaml